### PR TITLE
Post demo changes

### PR DESCRIPTION
**Pull request by laemtl**
Screenshot from 2021-06-15 12-26-58

### CODE REVIEW

File: src/jsx/Configuration.js [hunk 1]
```diff
@@ -25,103 +31,280 @@ const Configuration = (props) => {
...
-  const [lineFreq, setLineFreq] = useState('');
-  const [siteID, setSiteID] = useState('');
-  const [projectID, setProjectID] = useState('');
-  const [subProjectID, setSubProjectID] = useState('');
-  const [visitLabel, setVisitLabel] = useState('');
-  const [headerFields, setHeaderFields] = useState(null);
-  const [edfHeader, setHeader] = useState({
+  const state = {};
+  state.edfFiles = {};
+  [state.edfFiles.get, state.edfFiles.set] = useState([]);
```
Comment (maltheism) [inline src/jsx/Configuration.js:36]: Hi @laemtl, I'm unsure if you need to be initiating `state.edfFiles = {};` when the `useState({});` should be setting the initial type to an object. Similar for all the other lines below. Is that necessary if you combine everything into one giant `state` object?

**Comment by maltheism** [comment]
The code looks really nice!

File: python/libs/loris_api.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
+import json
+import requests
+import urllib
+
+class LorisAPI:
+    url = 'https://localhost/api/v0.0.4-dev/'
+    username = 'admin'
+    password = 'Pouliche123'
```
Comment (maltheism) [inline python/libs/loris_api.py:8]: Hi @laemtl, awesome work with all this cool stuff you've added! 

Eventually I'll try fixing the code for multiple windows of the GUI in react. I have some code for that commented out, it should already work but it was having problems when compiling to production. We could have a login button in the Configurations tab, with a similar position like the anonymize toggle you made but we can place the login button at the very top right (in the user's face) and where it opens a new window for user login credentials or maybe just an model like the loading functionality has. Then we could have three fields (url, username, password). So the EEG2BIDS tool can work on different LORIS instances if the user changes the URL.

**Comment by laemtl** [comment]
From @maltheism 

Hi @laemtl, awesome work with all this cool stuff you've added!

Eventually I'll try fixing the code for multiple windows of the GUI in react. I have some code for that commented out, it should already work but it was having problems when compiling to production. We could have a login button in the Configurations tab, with a similar position like the anonymize toggle you made but we can place the login button at the very top right (in the user's face) and where it opens a new window for user login credentials or maybe just an model like the loading functionality has. Then we could have three fields (url, username, password). So the EEG2BIDS tool can work on different LORIS instances if the user changes the URL.

File: src/jsx/Configuration.js (hunk 1 above)
Comment (laemtl) [reply src/jsx/Configuration.js:36]: I need to initialize state.edfFiles to {} in order to add the get and set keys without error/warning. useState set the initial value of the set key, not the parent dictionary.

Comment (laemtl) [reply src/jsx/Configuration.js:36]: I combine everything into a common state object in order to be able to loop on the keys to call setTask on all entries (previously the task keys were initially null).

**Comment by laemtl** [comment]
@maltheism, I totally agree with your suggestion, In fact, this is already planned. The current implementation is a shortcut for faster prototyping.

**Review by maltheism — APPROVED** [review]
Hi @laemtl, I believe we can merge this PR for now and add additional functionality to new PRs. Btw, I'm waiting on receiving the new laptop before I can test your latest code on a production build for Windows. I'll be using parallels to setup a test environment for the Windows 10 build. I should be receiving the new laptop on Wednesday. I'll likely have questions later in the week when I have everything setup and maybe early next week because of the Holiday including Summer Friday.

**Pull request merged**